src/Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GhDampingFunction.hpp

```diff
@@ -0,0 +1,63 @@
+// Distributed under the MIT License.
```

Comment:
`Gh` in file name is redundant with directory names. Maybe `ConstraintDamping` would be clearer too?